### PR TITLE
Sass grid

### PR DESCRIPTION
**Pull request by levimllr**
Styled the index grid. Want your thoughts about design before styling any more!

### CODE REVIEW

File: src/DetailPage.tsx [hunk 1]
```diff
@@ -29,10 +29,10 @@ const DetailPage:FunctionComponent<Props & RouteComponentProps<{ id: string}>> =
 
         if (onePokemon) {
             return (
-                <div className="page">
-                    { pokeNum > 1 ? <Card pokemon={pokemons[pokeNum - 2]} /> : null }
+                <div className="page grid-container display-items">
+                    { pokeNum > 1 ? <Card pokemon={pokemons[pokeNum - 2]} /> : <div></div> }
                     <DetailView pokemon={onePokemon} />
-                    { pokeNum < pokemons.length - 1 ? <Card pokemon={pokemons[pokeNum]} /> : null }
+                    { pokeNum < pokemons.length - 1 ? <Card pokemon={pokemons[pokeNum]} /> : <div></div> }
```
Comment (qualitydixon) [inline src/DetailPage.tsx]: Why the change to the empty div?

Comment (levimllr) [reply src/DetailPage.tsx]: Accommodates the grid when at the ends of the Pokemon numbers.
